routes/users.go

```diff
@@ -0,0 +1,38 @@
+package routes
+
+import (
+	"fmt"
+	"net/http"
+
+	"example.com/rest-api/models"
+	"github.com/gin-gonic/gin"
+)
+
+func signUp(context *gin.Context) {
+	var user models.User
+	err := context.ShouldBindJSON(&user)
+
+	if err != nil {
+		fmt.Println(err)
```

Comment:
it is not a good idea to use fmt.Println for logging, it will not show the timestamp.